knative/lister.go

```diff
@@ -12,8 +12,11 @@ import (
 )
 
 const (
-	labelKey   = "boson.dev/function"
-	labelValue = "true"
+	labelFunctionKey          = "function.knative.dev"
+	deprecatedFunctionKey     = "boson.dev/function"
```

Comment:
Minor nit - to be consistent with other naming.
```suggestion
	deprecatedLabelFunctionKey     = "boson.dev/function"
```

Reply:
good catch

Reply:
Fixed